DistributionPackages/Neos.NeosIo/Resources/Private/Scss/_Framework/_Fonts.scss

```diff
@@ -10,3 +10,21 @@ $font-path: "../Assets" !default;
   font-style: normal;
   font-weight: 400;
 }
```

Comment:
Don't we also need the regular italic font face? We can use italic in the ck editor.

Reply:
Yes! Thanks for the hint ❤️ Will be added with my next commit :)